lib/index.js

```diff
@@ -54,3 +56,8 @@ bedrock.events.on('bedrock.init', () => {
 function _invalidStateError() {
   return new BedrockError('The agent is not ready.', 'InvalidStateError');
 }
+
+// For testing purposes only
+function setHttpsAgent({httpsAgent}) {
```

Comment:
Sorry if I was unclear before, this should still be a private function, so `_setHttpsAgent`.

Reply:
Updated b533851